metadata.json

```diff
@@ -47,19 +47,19 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">=4.24.0 <6.0.0"
+      "version_requirement": ">=4.24.0 <8.0.0"
```

Comment:
Latest version of `stdlib` is `6.1.0`, according to [puppetforge](https://forge.puppet.com/puppetlabs/stdlib). Would it be safer to have `<7.0.0` as the requirement, in case version 7 of this lib breaks something?

Reply:
🙇